packages/contracts-bedrock/src/safe/TimelockGuard.sol

```diff
@@ -451,10 +452,12 @@ abstract contract TimelockGuard is BaseGuard {
         // Record the calling Safe
         Safe callingSafe = Safe(payable(msg.sender));
 
-        // Check that the contract is at least version 1.3.0
-        // Prior to version 1.3.0, checkSignatures() was not exposed as a public function, so we need to check the
-        // version otherwise the safe will be bricked.
-        if (SemverComp.lt(callingSafe.VERSION(), "1.3.0")) {
+        // Check that the safe contract is version 1.3.x or 1.4.x
+        // Prior to version 1.3.0, checkSignatures() was not exposed as a public function, on 1.5.0
+        // `encodeTransactionData` was removed from the safe, and the `isValidSignature` function
+        // signature changed. If the guard were to be enabled for an unsupported version the safe
+        // would be rendered permanently unusable.
+        if (SemverComp.lt(callingSafe.VERSION(), "1.3.0") || SemverComp.gte(callingSafe.VERSION(), "1.5.0")) {
```

Comment:
There is an inconsitency between TimelockGuard.sol and LivenessModule.sol 
```
if (SemverComp.lt(callingSafe.VERSION(), "1.3.0") || SemverComp.gte(callingSafe.VERSION(), "1.5.0")) {
```
vs 
```
if (SemverComp.lt(callingSafe.VERSION(), "1.3.0") || SemverComp.gte(callingSafe.VERSION(), "1.6.0")) {
```

`1.5.0` vs `1.6.0`, I want to make sure this is desired, it would be better for consistency to have the same version checking, is that possible? 

I wants to avoid the fact, you can have the same safe version that might be not working for either the `LivenessModule2` or the `TimelockGuard` and this might be confusing imo.